asset-transfer-basic/application-java/src/test/java/application/java/TestApp.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Running tests: 
+// gradle test --tests TestApp.start
+// gradle test --tests TestApp.CreateAssetTest
+// gradle test --tests TestApp.ReadAssetTest
+// gradle test --tests TestApp.DeleteAssetTest
+// gradle test --tests TestApp.UpdateAssetTest
+// gradle test --tests TestApp.TransferAssetTest
+// gradle test --tests TestApp.GetAllAssetsTest
+
+package application.java;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+
+public class TestApp {
+
+	@Test
+	public void start() throws Exception {
+		EnrollAdmin.main(null);
+		RegisterUser.main(null);
+		ClientUtilities.InitLedger();
+	}
+	// Creates asset7, modify the parameters to create other assets
+	@Test
+	public void CreateAssetTest() throws Exception {
+		ClientUtilities.CreateAsset("asset7", "magenta", "20", "kenysha", "800");
+	}
+
+	// Reads asset4, modify the parameters to read other assets
+	@Test
+	public void ReadAssetTest() throws Exception {
+		ClientUtilities.ReadAsset("asset4");
```

Comment:
If we go with unit test to demo the app, then we should add assert statements

Reply:
I switched directions and removed the unit tests, a main file will be used instead to showcase the app.